Source/driver/Castro.cpp

```diff
@@ -340,6 +343,14 @@ Castro::read_params ()
 	pp.add("ppm_trace_sources",ppm_trace_sources);
       }
 
+    // fourth order implies do_ctu=0
+    if (fourth_order == 1 && do_ctu == 1)
+      {
+	if (ParallelDescriptor::IOProcessor())
+	    std::cout << "WARNING: fourth_order requires do_ctu = 0.  Resetting do_ctu = 0" << std::endl;
```

Comment:
I would prefer this to throw an error rather than change the value of do_ctu.

Reply:
fair enough... I did this mostly for when I was debugging